docs/ci/demo

```diff
@@ -174,42 +174,12 @@ popper env mypipeone | grep 'host'
 
 # popper add
 init_test
-popper add popperized/quiho-popper/single-node
-test -d pipelines/single-node
-test -f pipelines/single-node/datapackage/quiho/datapackage.json
-test -f pipelines/single-node/datapackage/quiho/machines.csv
-test -f pipelines/single-node/datapackage/quiho/results.csv
-test -f pipelines/single-node/geni/release.py
-test -f pipelines/single-node/geni/request.py
-test -f pipelines/single-node/setup/cloudlab.sh
-test -f pipelines/single-node/setup/travis.sh
-test -f pipelines/single-node/results/postprocess.sh
-test -f pipelines/single-node/results/visualize.ipynb
-test -f pipelines/single-node/results/visualize.sh
-test -f pipelines/single-node/results/figures/apps_variability.pdf
-test -f pipelines/single-node/results/figures/mariadb-innodb-vs-memory.pdf
-test -f pipelines/single-node/results/figures/corrmatrix.pdf
-test -f pipelines/single-node/results/figures/pca-var-reduction.pdf
-test -f pipelines/single-node/results/figures/corrmatrix_underfit.pdf
-test -f pipelines/single-node/results/figures/redis-set_underfit.pdf
-test -f pipelines/single-node/results/figures/four.pdf
-test -f pipelines/single-node/results/figures/stream-nadds-behavior.pdf
-test -f pipelines/single-node/results/figures/hpccg.pdf
-test -f pipelines/single-node/results/figures/stream-nadds.pdf
-test -f pipelines/single-node/results/figures/mariadb-innodb-regression.pdf
-test -f pipelines/single-node/results/figures/stressng_variability.pdf
-test -f pipelines/single-node/results/previous/2017-06-07T0501.tgz
-test -f pipelines/single-node/results/previous/20170821-120356-88f97bd.tgz
-test -f pipelines/single-node/results/previous/2017-06-07T0502.tgz
-test -f pipelines/single-node/results/previous/2017-10-20-mysql-regression.tgz
-test -f pipelines/single-node/results/previous/2017-06-07T0503.tgz
-test -f pipelines/single-node/run.sh
-test -f pipelines/single-node/teardown.sh
-test -f pipelines/single-node/wf.png
-test -f pipelines/single-node/post-run.sh
-test -f pipelines/single-node/validate.sh
-test -f pipelines/single-node/setup.sh
-test -f pipelines/single-node/vars.yml
+popper add popperized/lesson/co2-emissions
+test -d pipelines/co2-emissions
+test -f pipelines/co2-emissions/README.md
+test -f pipelines/co2-emissions/run.sh
+test -f pipelines/co2-emissions/setup.sh
+test -f pipelines/co2-emissions/validate.sh
 
 # info command
 popper info popperized/quiho-popper/single-node | grep 'Github Url'
```

Comment:
The `info` subcommand searches the local storage to find the pipeline. Since the single-node pipeline is not added,
this test will fail.

this needs to be changed to `popper info popperized/lesson/co2-emissions | grep 'Github Url' `

Reply:
nice catch. I've updated the tests so this is properly referencing `co2-emissions`